dependencies/influxdb/http.go

```diff
@@ -441,3 +457,59 @@ func (h seriesCardinalityHttpReader) Read(ctx context.Context, f func(flux.Table
 	}
 	return h.Query(ctx, f, &file, h.Bounds.Now, mem)
 }
+
+type httpWriter struct {
+	writer      *api.WriteAPIImpl
+	errChan     <-chan error
+	latestError chan error
+}
+
+func NewHttpWriter(writer *api.WriteAPIImpl) (*httpWriter, error) {
+	w := &httpWriter{
+		writer:      writer,
+		errChan:     writer.Errors(),
+		latestError: make(chan error, 1),
+	}
+	go func() {
```

Comment:
The error handling here is a bit ugly, there is an open issue for the client to improve this with a v3 of the client.